packages/components/src/components/tile/_tile.scss

```diff
@@ -105,11 +105,15 @@
 
   .#{$prefix}--tile__chevron {
     position: absolute;
-    right: 0.5rem;
-    bottom: 0.5rem;
+    right: $carbon--spacing-05;
+    bottom: $carbon--spacing-05;
+    display: flex;
+    flex-direction: row;
```

Comment:
This is the default property, so we can probably remove this line

Reply:
Ah, yeah. Good call.